.github/workflows/autobuild.yml

```diff
@@ -94,14 +94,22 @@ jobs:
             # Jamulus.pro needs to count git history length for android versioning:
             checkout_fetch_depth:   '0'
 
-          - config_name:            Linux .deb (artifacts+codeQL)
+          - config_name:            Linux .deb x86_64 (artifacts+codeQL)
```

Comment:
The debian name (and our .deb artifact name) is amd64. So we could name this amd64 instead of x86_64 to make this consistent.
Counter-point: amd64 is unintuitive for people who don't want to think about CPU architectures. :)

Reply:
I think it's better to be consistent with the platform naming conventions - as that's the target audience.  We're likely not the only software they'll be dealing with, so doing something unexpected will cause more confusion than not.

Reply:
Ok. Changed to amd64 now.